contrib/python/src/python/pants/contrib/python/checks/tasks/python_eval.py

```diff
@@ -104,6 +104,9 @@ def _compile_target(self, target):
       if isinstance(target, PythonBinary):
         source = 'entry_point {}'.format(target.entry_point)
         components = target.entry_point.rsplit(':', 1)
+        if not all([x.strip() for x in components]):
+          raise TaskError('Invalid entry point {} for target {}'.format(
```

Comment:
This error could be more specific probably... the issue here is an empty component?